File: myRPG/HealthPotion.cpp
#include "HealthPotion.h"

HealthPotion::HealthPotion(int count){
	name = "Health potion";
	description = "Red mixture that makes you fly. Heals for 10 hp";
	quantity = count;
	maxDurability = 1;
	durability = 1;
	itemSlot = ItemSlot::POTIONS;
	requirements = std::make_shared<Requirements>();
	requirements->level = 2;
}

HealthPotion::~HealthPotion(){
}

File: myRPG/Health potion.lua
--lua script
item = {
	name = "Health potion",
	description = "Red mixture that makes you fly. Heals for 10 hp",
	quantity = 1,
	attackValue = 0,
	armorValue = 0,
	usable = 1,
	maxDurability = 1,
	durability = 1,
	itemSlot = "POTIONS",
	Abilities = {
		"Heal",
	},
	Resistance = {
		cold = 0,
		fire = 0,
		electricity = 0,
		water = 0,
	},
	Requirements = {
		class = { 
			"Mage",
			"Warrior",
		},
		level = 2,
	},
}
